tests/explicit-nulls/neg/i17467.check

```diff
@@ -0,0 +1,31 @@
+-- [E007] Type Mismatch Error: tests/explicit-nulls/neg/i17467.scala:4:22 ----------------------------------------------
+4 |    val a2: a1.type = null // error
+  |                      ^^^^
+  |                      Found:    Null
+  |                      Required: (a1 : String)
+  |                      Note that implicit conversions were not tried because the result of an implicit conversion
+  |                      must be more specific than (a1 : String)
```

Comment:
An unfortunate error message, seeing how `a1 : String` is already a fairly specific type...  (not a blocker imo)

Reply:
I agree, but that error message has nothing to do with nullability, AFAICT. Has it?

Reply:
I think it's possible, given `isUnderSpecifiedArgument` ends up checking if the argument type `isRef(defn.NullClass)`.